### PR TITLE
Update html

### PR DESCRIPTION
**Pull request by DewGew**
This fix makes the tabs (switches, scenes, temperatures e.t.c.) to have same behavior when creating themes.

### CODE REVIEW

File: www/views/utility.html [hunk 1]
```diff
@@ -215,7 +215,7 @@
     </form>
 </div>
 
-<div>
+<div class="container">
 	<gz-back-to-top></gz-back-to-top>
 	<div id="utilitycontent" class="container"></div>
```
Comment (gizmocuz) [inline www/views/utility.html]: i ithink the 'class="container"' needs to be removed here ?

**Comment by gizmocuz** [comment]
Thanks for the PR, i have one question (see above), there is a double class="container" in utility.html with this patch, and i think the last one should be removed

**Comment by DewGew** [comment]
I have Edit that one now

**Comment by gizmocuz** [comment]
Thanks!!

**Pull request merged**